Docker Desktop 3.2.0 release notes (#12388)
docker/docker.github.io: docker-for-windows/release-notes.md

> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
{: .important }

## Docker Desktop Community 3.2.0
2021-03-NN

> [Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe)

### New

> - Windows 16299 is marked as deprecated; it is not supported going forward in releases

Reviewer: We used to put this in a `Deprecation` section, e.g. last time it was:
```
    Docker Desktop cannot be installed on Windows 1703 (build 15063) anymore.
```

### Upgrades

- [Docker Engine 20.10.4](https://docs.docker.com/engine/release-notes/#20104)
- [Docker Compose 1.28.4](https://github.com/docker/compose/releases/tag/1.28.4)
- [Compose CLI v1.0.8](https://github.com/docker/compose-cli/tree/v1.0.8)
- [Docker Hub Tool v0.3.0](https://github.com/docker/hub-tool/releases/tag/v0.3.0)
- [QEMU 5.0.1](https://wiki.qemu.org/ChangeLog/5.0)
- [Amazon ECR Credential Helper v0.5.0](https://github.com/awslabs/amazon-ecr-credential-helper/releases/tag/v0.5.0)
- [Alpine 3.13](https://alpinelinux.org/posts/Alpine-3.13.0-released.html)
- [Kubernetes 1.19.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.7)
- [Go 1.16](https://golang.org/doc/go1.16)

### Deprecation

- Docker Desktop cannot be installed on Windows 1709 (build 16299) anymore.

### Bug fixes and minor changes

- Fixed an issue when port forwarding multiple ports with an IPv6 container network. Fixes [docker/for-mac#5247](https://github.com/docker/for-mac/issues/5247)
- Fixed a regression where `docker load` could not use an xz archive anymore. Fixes [docker/for-win#10364](https://github.com/docker/for-win/issues/10364)

> - Fixed WSL 2 backend shutdown process interfering with Windows shutdown. [docker/for-win#5825](https://github.com/docker/for-win/issues/5825) [docker/for-win#6933](https://github.com/docker/for-win/issues/6933) [docker/for-win#6446](https://github.com/docker/for-win/issues/6446)

Reviewer 2: ```suggestion
- Fixed an issue that caused the WSL 2 backend shutdown process to interfere with Windows shutdown. [docker/for-win#5825](https://github.com/docker/for-win/issues/5825) [docker/for-win#6933](https://github.com/docker/for-win/issues/6933) [docker/for-win#6446](https://github.com/docker/for-win/issues/6446)
```

Author: Thanks, I'll update it.

- Fixed creds store using `desktop.exe` from WSL 2. Fixes [docker/compose-cli#1181](https://github.com/docker/compose-cli/issues/1181)
- Fixed navigation issue between apps/containers screens [docker/for-win#10160](https://github.com/docker/for-win/issues/10160#issuecomment-764660660)
- Fixed container instance view with long container/image name [docker/for-win#10160](https://github.com/docker/for-win/issues/10160)
- Fixed an issue when binding ports on specific IPs. Notice: It may now take a bit of time before an inspect shows the open ports. Fixes [docker/for-win#10008](https://github.com/docker/for-win/issues/10008)

> - Fixed scrolling on container detail screen [docker/for-win#10160](https://github.com/docker/for-win/issues/10160)

Reviewer 2: ```suggestion
- Fixed an issue related to scrolling on the Container view screen. [docker/for-win#10160](https://github.com/docker/for-win/issues/10160)
```

> - Fixed a bug where an image deleted from the dashboard would remain displayed

Reviewer 2: ```suggestion
- Fixed an issue where an image deleted from the Docker dashboard was still displayed on the **Images** view.
```

Author: Yes!
